Resources/Prototypes/_Mono/Shipyard/jug.yml

```
@@ -0,0 +1,53 @@
+- type: vessel
+  id: Jug
+  parent: BaseVessel
+  name: UNJ Jug
+  description: A science vessel that requires some setup. Great for Science-Salvage crews!
+  price: 33000
+  category: Medium
+  group: Shipyard
+  shuttlePath: /Maps/_Mono/Shuttles/jug.yml
+  guidebookPage: null
+  class:
+  - Cargo
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)